iocore/net/quic/QUICCongestionController.cc

```diff
@@ -87,11 +87,11 @@ QUICCongestionController::on_packet_acked(QUICPacketNumber acked_packet_number,
 }
 
 void
-QUICCongestionController::on_packets_lost(std::map<QUICPacketNumber, PacketInfo &> lost_packets)
+QUICCongestionController::on_packets_lost(std::map<QUICPacketNumber, PacketInfo *> lost_packets)
```

Comment:
Should this `std::map` be passed by reference instead of by value?

Reply:
Thanks. will fix it.